### PR TITLE
android: Upgrade the Gradle plugin to get the CMake output

### PR DESCRIPTION
**Pull request by Orphis**

### CODE REVIEW

**Comment by Orphis** [comment]
We probably want to wait for the Gradle plugin 2.3 to be out of beta first.
The major change in there is that we can see the output of the CMake build real time.

**Comment by degasus** [comment]
Yeah, please wait for a stable release. Feel free to also drop the android-cmake-installer reference in https://github.com/dolphin-emu/dolphin/blob/master/AndroidSetup.md

**Comment by Orphis** [comment]
I haven't tried to install CMake with the new plugin, so I'll try it first :)

**Comment by PEmu1** [comment]
2.3 is out now.

File: Source/Android/app/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.android.application'
 android {
     // Leanback support requires >22
     compileSdkVersion 24
-    buildToolsVersion '24.0.3'
+    buildToolsVersion '25.0.0'
```
Comment (degasus) [inline Source/Android/app/build.gradle]: May you also update the version of the support libraries?

**Comment by ligfx** [comment]
This was done in #5013

**Comment by degasus** [comment]
Yeah, sorry. I was faster :P

However, thanks for the patch.

**Pull request closed without merging**